src/server/handlers/github-auth.js

```diff
@@ -3,7 +3,7 @@ import logging from '../logging';
 
 import { conf } from '../helpers/config';
 
-const AUTH_SCOPE = 'admin:repo_hook';
+const AUTH_SCOPE = 'admin:repo_hook'; // another scope (like 'public_repo') may be needed to list organisations' repositories
```

Comment:
Right. I just checked what Travis asks for, and according to my settings page they use:

```
 Read org and team membership
 Access commit status
 Access deployment status
 Access user email addresses (read-only)
 Write repository hooks
```

Not sure how exactly it translates to scope values, but we have a list to compare to when we want to change these.